clients/ui/frontend/src/__mocks__/mockCatalogFilterOptionsList.ts

```diff
@@ -52,6 +54,15 @@ export const mockCatalogFilterOptionsList = (
       type: 'string',
       values: ['GPU', 'CPU', 'TPU', 'FPGA'],
     },
+    [ModelCatalogStringFilterKey.USE_CASE]: {
+      type: 'string',
+      values: [
+        UseCaseOptionValue.CHATBOT,
+        UseCaseOptionValue.CODE_FIXING,
+        UseCaseOptionValue.LONG_RAG,
+        UseCaseOptionValue.RAG,
+      ],
+    },
```

Comment:
I'm not seeing any results when I filter in mock mode based on these, we should follow up to add some of these values to the mock data in the BFF